test/new/db/anal/x86_64

```diff
@@ -1089,8 +1089,8 @@ EXPECT=<<EOF
 |  | | |    0x00003cda      mov byte [rip + 0x1d33f], 1                ; [0x21020:1]=0
 | ========< 0x00003ce1      jmp 0x3cf9
 |  | | |    ; CODE XREF from main @ 0x3cb2
-|  | `----> 0x00003ce3      mov esi, 5                                 ; int64_t arg2
-|  |   |    0x00003ce8      xor edi, edi                               ; int64_t arg1
+|  | `----> 0x00003ce3      mov esi, 5
+|  |   |    0x00003ce8      xor edi, edi
```

Comment:
imho this is a regression

Reply:
relevant: https://github.com/radareorg/radare2-regressions/pull/2007